wiki/Beatmaps/Packs/en.md

```diff
@@ -0,0 +1,48 @@
+# Beatmaps Packs
+
+_Main page: [Beatmaps](/wiki/Beatmaps)_
+
+Beatmaps packs, as the title suggests, are a `.zip` files containing beatmaps.
+The name of the pack is based on what they contain (e.g. `Approved Beatmap Pack #7` would only contain [approved](#approved) beatmaps).
+It is worth noting that the most packs are:
+
+- Roughly either ~100MB (Taiko/osu!mania) or ~200MB (-/Approved)
+- Always contain videos (if included)
+- The beatmaps are either in the form of `.zip` or `.osz` file
+
+## Categories
+
+Beatmaps packs split into 4 categories:
+
+| Category | Beatmap types |
+| -------- | ------------- |
+| Standard | Ranked or Approved songs |
+| Chart | Fortnightly Chart challenge songs |
+| Theme | General theme of the songs |
+| Artist/Album | Based on Artist/Album |
+
+## Obtaining Every Beatmap
+
```

Comment:
That header doesnt make any sense; no information about how to obtain every beatmap